app/frontend/wallet/blockchain-explorer.ts

```diff
@@ -372,16 +409,16 @@ const blockchainExplorer = (ADALITE_CONFIG) => {
     const url = `${
       ADALITE_CONFIG.ADALITE_BLOCKCHAIN_EXPLORER_URL
     }/api/account/info/${stakingKeyHashHex}`
-    const response = await request(url)
+    const response: AccountInfoResponse = await request(url)
```

Comment:
Maybe someone more with more typescript can say, but personally I would not type the `const`, I would type the return statement of the function. I get that `request` returns `Promise<any> and there is not much to do about that so we have to type the const, but still I would type the return statement on all functions as well. But as I said, someone more experienced could comment :D

Reply:
Check the other comment